obstacle_radii: [0.1, 0.2, 0.3]
obstacle_heights: [7, 10, 3]
obstacle_xs: [-5, -3, -1]
obstacle_ys: [-1, -3, -5]